cmd/update-main.go

```diff
@@ -62,15 +63,22 @@ EXAMPLES:
 `,
 }
 
-const releaseTagTimeLayout = "2006-01-02T15-04-05Z"
-
-const minioReleaseURL = "https://dl.minio.io/server/minio/release/" + runtime.GOOS + "-" + runtime.GOARCH + "/"
+const (
+	minioReleaseTagFormat = "2006-01-02T15-04-05Z"
```

Comment:
the name `minioReleaseTagFormat` looks like a release tag format however the const is a time format portion in the release tag.

golang calls time format as layout